tests/Browser/ItemsFormTest.php

```diff
@@ -22,4 +22,16 @@ public function test_can_enter_list_of_item_ids()
                     ->assertInputValue('@items-input', "Q1\nQ2");
         });
     }
+
+    public function test_can_submit_list_of_item_ids()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->visit(new HomePage)
+                    ->keys('@items-input', 'Q1', '{return_key}', 'Q2')
+                    ->press('button')
+                    ->waitFor('div.results')
```

Comment:
Instead of referring to the classes directly, I think It's best to add shortcuts for them in the `elements` method of the Page object we use. That way, if anything changes in the client side app, we will only need to update it in one place.